meetings/submissions/views.py

```diff
@@ -61,30 +62,23 @@ def create(self, request, *args, **kwargs):
                 }
 
                 response = requests.post(
-                    self.node_url,
-                    data=json.dumps(node),
-                    headers={
-                        'Authorization': 'Bearer {}'.format(osf_token),
-                        'Content-Type': 'application/json; charset=UTF-8',
-                        'Accept': 'application/json, text/*'
-                    }
+                        self.node_url,
+                        data=json.dumps(node),
+                        headers={
+                            'Authorization': 'Bearer {}'.format(osf_token),
+                            'Content-Type': 'application/json; charset=UTF-8'
+                        }
                 )
 
-                obj = response.json()
+                resObj = response.json()
```

Comment:
`obj` is fine in this situation, `resObj` follows camelCase which is appropriate for js, but in python stick with snake_case for variables. So `response_object` would be good since it is descriptive.